client/my-sites/customer-home/cards/tasks/podcasting/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { TASK_PODCASTING } from 'calypso/my-sites/customer-home/cards/constants';
+import podcastingIllustration from 'calypso/assets/images/customer-home/illustration--task-podcasting.svg';
+
+const Podcasting = () => {
+	const translate = useTranslate();
+
+	return (
+		<Task
+			title={ translate( 'Grow your audience with a podcast' ) }
+			description={ translate(
+				`Easily turn your blog into a podcast with Anchor — the world's biggest podcasting platform.`
+			) }
+			actionText={ translate( 'Get started' ) }
+			actionUrl={ `https://anchor.fm` }
```

Comment:
Is this a temporary link? Similarly to the Stripe connection process on Earn blocks, I think this should redirect to a URL where users can connect their Anchor.fm account to their WP.com account and then they are redirected back to WP.com. But probably that process is still not implemented, in that case the temporary link is fine, but maybe we can include a TODO comment linking to the pending issue.

Reply:
I followed the description here: 310-gh-dotcom-manage

I agree -- the action URL as it stands doesn't make sense to me. I'll add a TODO